guides/common/assembly_configuring-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet_{context}"]
+endif::[]
+= Configuring Puppet
+
+:context: assembly_configuring-puppet
+
+[role="_abstract"]
```

Comment:
```suggestion
```

Reply:
This is a template-given standard that we use for metadata in downstream publishing, therefore it will stay.
See https://github.com/theforeman/foreman-documentation/pull/890#discussion_r760410889